src/tests/test_utils/common_test_utils/include/common_test_utils/type_ranges.hpp

```diff
@@ -0,0 +1,187 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <map>
+#include <vector>
+
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "openvino/core/node.hpp"
+#include "openvino/core/type/element_type_traits.hpp"
+
+namespace ov {
+namespace test {
+namespace utils {
+
+static ov::test::utils::InputGenerateData get_range_by_type(ov::element::Type temp_type, uint32_t max_range_limit = testing::internal::Random::kMaxRange) {
+    double min_start = 0 - (int32_t)round(max_range_limit / 2);
```

Comment:
Please, use `static_cast` instead of c style cast

Reply:
changed